### ISSUE TITLE
net.minecraft.client.renderer.texture.NativeImage.PixelFormat changes

### ISSUE THREAD

**Issue by Lanse505** [MinecraftForge/MCPConfig#135]
Changes to NativeImage.PixelFormat enum.

I've checked and this seems to be a MCPConfig change and not a srg mapping changeable thing.
![bild](https://user-images.githubusercontent.com/1495068/103781240-5fc2f380-5036-11eb-9529-b7ff0374cd1c.png)

``net.minecraft.client.renderer.texture.NativeImage.PixelFormat#RGBA -> 
net.minecraft.client.renderer.texture.NativeImage.PixelFormat#ABGR``

``net.minecraft.client.renderer.texture.NativeImage.PixelFormat#RGB ->
net.minecraft.client.renderer.texture.NativeImage.PixelFormat#BGR``

Basically what seems to have occured is that someone has mapped these based of "logical" color sense and named them ``RBGA`` and ``RGB``

While Minecraft seems to be using a legacy standard due to legacy code and uses ``ABGR`` and ``BGR`` as it's standard.

**Comment by Lanse505** [comment]
This was reported by HellfirePvP and confirmed by Buuz135

**Comment by Lanse505** [comment]
After some talk with @Cyborgmas in discord.
Creating a new system to remap the enums is pointless.

However establishing something like a javadoc on the inner-class might be useful for tracking?

**Comment by LexManos** [comment]
Enums can not be remapped because that breaks the enum contract.
They contain the name of the field as the name of the enum.
Which is why they are not assigned a SRG name. But yes, they should be able to get javadocs.
With the new indexing system they are assigned their own unique ids so if some other crowdsourcing project comes along to build javadocs they can be injected.
